.github/workflows/release.yaml

```diff
@@ -0,0 +1,121 @@
+name: Create a release
+
+on:
+  # Trigger a stable version release via GitHub's UI, with the ability to specify the type of release.
+  workflow_dispatch:
+    inputs:
+      release_type:
+        description: Release type
+        required: true
+        type: choice
+        default: auto
+        options:
+          - auto
+          - custom
+          - patch
+          - minor
+          - major
+      custom_version:
+        description: The custom version to bump to (only for "custom" type)
+        required: false
+        type: string
+        default: ""
+
+jobs:
+  release_metadata:
+    name: Prepare release metadata
+    runs-on: ubuntu-latest
+    outputs:
+      version_number: ${{ steps.release_metadata.version_number }}
+      tag_name: ${{ steps.release_metada.tag_name }}
+      changelog: ${{ steps.release_metadata.changelog }}
+      release_notes:: ${{ steps.release_metadata.release_notes }}
+    if: "!startsWith(github.event.head_commit.message, 'docs') && !startsWith(github.event.head_commit.message, 'ci')"
+    steps:
+      - uses: apify/workflows/release-metadata@release-metadata-action
+        name: Prepare release metadata
+        id: release_metadata
+        with:
+          release_type: ${{ inputs.release_type }}
+          custom_version: ${{ inputs.custom_version }}
+
+  lint_check:
+    name: Lint check
+    uses: apify/workflows/.github/workflows/python_lint_check.yaml@main
+
+  type_check:
+    name: Type check
+    uses: apify/workflows/.github/workflows/python_type_check.yaml@main
+
+  unit_tests:
+    name: Unit tests
+    uses: apify/workflows/.github/workflows/python_unit_tests.yaml@main
```

Comment:
On a manual release, this really seems redundant. Also, see #469

Reply:
Currently, it is not redundant, since we don't have any additional trigger of code checks for the master (only for PRs).